frontend/src/App.js

```diff
@@ -3,12 +3,20 @@ import './App.css';
 import {BrowserRouter as Router, Switch, Route} from 'react-router-dom'
 import Navbar from './components/layout/Navbar'
 import Footer from './components/layout/Footer';
+import Card from './components/layout/Card';
 
 function App() {
   return (
 	<Router>
 		<div className="App">
 			<Navbar/>
+			<Card
```

Comment:
Instead of adding card here, create another component which has state of all cards( using Hooks) . Use that and run a map on Card ( atleast 3 items). Display that component here. ( Name it "Cards")

Reply:
![image](https://user-images.githubusercontent.com/43892590/107032892-882b4280-67da-11eb-8641-3f593ba522a8.png)

Reply:
Yes, make sure its responsive. Commit the changes as well.

Reply:
@ashishnagpal2498 Done. Pushed the latest code.